### PR TITLE
You can access your inventory when lying down

### LINKED ISSUE #84373: Can no longer access inventories while on the ground
## Issue Summary

Attempting to access your bag while prone gives you 'You don't have the mobility for this!' message.
You can still access your bag as normal if you had it open before laying down

## Round ID:
[231200](https://scrubby.melonmesa.com/round/231200)

## Testmerges:
https://github.com/tgstation/tgstation/pull/83227
https://github.com/tgstation/tgstation/pull/84203
https://github.com/tgstation/tgstation/pull/84083

## Reproduction:

> lay on ground
> attempt to access bag
> BAG STUCK
> die

### PR DESCRIPTION
**Pull request by SyncIt21**
## About The Pull Request
- Fixes #84373

## Changelog
:cl:
fix: you can access your inventory when lying down
/:cl:

### CODE REVIEW

**Comment by Higgin** [comment]
Any chance this could get the FUCK/speedmerge label?

File: code/datums/storage/storage.dm [hunk 1]
```diff
@@ -962,7 +962,7 @@ GLOBAL_LIST_EMPTY(cached_storage_typecaches)
 		show_contents(to_show)
 		return FALSE
 
-	if(!isliving(to_show) || !to_show.can_perform_action(parent))
+	if(!isliving(to_show) || !to_show.can_perform_action(parent, ALLOW_RESTING | NEED_HANDS | FORBID_TELEKINESIS_REACH))
```
Comment (MrMelbert) [inline code/datums/storage/storage.dm:965]: The `needs_hands` check is going to break silicons opening bags I think?

Comment (MrMelbert) [reply code/datums/storage/storage.dm:965]: Which is/was 100% possible prior

Comment (SyncIt21) [reply code/datums/storage/storage.dm:965]: Sure I'll remove it

**Pull request merged**